src/run_sim.jl

```diff
@@ -18,7 +18,7 @@ function run_sim(method, args_multicopter, multicopter::FlightSims.Multicopter,
     )
     pos_cmd_func(t) = traj_des(t)
     mkpath(dir_log)
-    file_path = joinpath(dir_log, lpad(string(case_number), 4, '0') * "_" * TRAJ_DATA_NAME)
+    file_path = joinpath(dir_log, lpad(string(case_number), 4, '0') * "_τ_" * string(fdi.τ) * "_" * TRAJ_DATA_NAME)
```

Comment:
```suggestion
    file_path = joinpath(dir_log, lpad(string(case_number), 4, '0') * "_" * TRAJ_DATA_NAME)
```

여기는 optional; 기존의 디렉토리명이 바뀌는게 불편하면 이것은 어떨까.. 정도로?